plugins/active-directory/tests/__mocks__/Logger.ts

```diff
@@ -0,0 +1,14 @@
+import { Logger } from '@verdaccio/types';
+
+const logger: Logger = {
+  warn: jest.fn(),
+  error: jest.fn(),
+  // fatal: jest.fn(),
```

Comment:
I would recommend this, multiple times I had to add it myself.

Reply:
Do you mean uncomment this line? If that, it's done :wink: